templates/oe-corporate-blocks-footer.html.twig

```diff
@@ -4,18 +4,50 @@
  * Default theme implementation for rendering corporate blocks footer.
  *
  * Available variables:
- * - corporate_footer: Array with structure compatible to ECL ec-components-footer component.
+ * - custom_footer: Array (optional) with structure of custom footer compatible to ECL ec-components-footer component.
+ *   - site_identity: Site identity for custom footer.
+ *   - social_links: Social media links for custom footer.
+ *   - other_links: Other links for custom footer.
+ * - corporate_footer: Array with structure of corporate footer compatible to ECL ec-components-footer component.
  *   - about_ec: Data about EC for corporate footer.
  *   - social_media: Social media links for corporate footer.
  *   - about_eu: Data about EU for corporate footer.
+ *   - bottom_links: Data about EU for corporate footer.
  *
  * @see template_preprocess_corporate_blocks_footer()
  *
  * @ingroup themeable
  */
 #}
 
-<div class="footer__corporate-top">
+{%  if custom_footer is not empty %}
```

Comment:
According to my comment above "We need to always show the three elements?".
If that's correct, then we should need only the condition above: `if custom_footer is not empty`.
So these shouldn't be needed because we already ensured we have all the 3.
`if custom_footer.social_links is not empty`
`if custom_footer.other_links is not empty`
`if custom_footer.other_links is not empty`